package.json

```diff
@@ -23,7 +23,7 @@
   },
   "homepage": "https://github.com/segment-integrations/analytics.js-integration-google-tag-manager#readme",
   "dependencies": {
-    "@segment/analytics.js-integration": "^2.1.0",
+    "@segment/analytics.js-integration": "^3.0.0",
```

Comment:
Could you make this ^3.2.0, just to be sure it can only be fulfilled when the assumesPageView fix is included